public/end.json

```
@@ -0,0 +1,38 @@
+[
+    {
+        "message": "何も対策をしなかったあなたは、強風に煽られて飛んできた物に頭を強く打ち、死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message": "走って逃げたあなただったが、台風に敵うはずもなく、追いつかれて空高く吹き上げられてしまい、死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message":"台風を見物していたあなただったが、強風に吹き飛ばされて全身を強く打ち付け、死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message": "車で避難を始めたあなただったが、水かさが上がったことで車内に取り残され、死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message":"川の近くを通って避難していたあなただったが、氾濫した川の流れに押し流され、死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message":"斜面の近くを通って避難していたあなただったが、土砂崩れに巻き込まれて死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message":"来た道を引き返したあなただったが、一次戻ったことで台風の勢いが増してしまい、濁流にのみ込まれて死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message":"姿勢が高いまま移動してしまったあなたは、強風に吹き飛ばされ、地面に強く体を打ち付けて死んでしまった...",
+        "isGoodEnd": false
+    },
+    {
+        "message":"あなたは数々の苦難を乗り越えて避難所へたどり着いた。おめでとう！",
+        "isGoodEnd": true
```

```suggestion
        "message": "川の近くを通って避難していたあなただったが、氾濫した川の流れに押し流され、死んでしまった...",
        "isGoodEnd": false
    },
    {
        "message": "斜面の近くを通って避難していたあなただったが、土砂崩れに巻き込まれて死んでしまった...",
        "isGoodEnd": false
    },
    {
        "message": "来た道を引き返したあなただったが、一次戻ったことで台風の勢いが増してしまい、濁流にのみ込まれて死んでしまった...",
        "isGoodEnd": false
    },
    {
        "message": "姿勢が高いまま移動してしまったあなたは、強風に吹き飛ばされ、地面に強く体を打ち付けて死んでしまった...",
        "isGoodEnd": false
    },
    {
        "message": "あなたは数々の苦難を乗り越えて避難所へたどり着いた。おめでとう！",
        "isGoodEnd": true
```

プロパティ名の後ろの`:`と値の間にはスペース入れてほしい気がします。